sale_commission/models/account_invoice.py

```diff
@@ -62,26 +62,26 @@ class AccountInvoiceLine(models.Model):
     _inherit = "account.invoice.line"
 
     @api.model
-    def _default_agents(self):
+    def create(self, vals):
```

Comment:
You must not replace the default. It's there for something: modify in screen the default commission values if needed.